src/Tools/dotnet-monitor/Program.cs

```diff
@@ -79,13 +84,18 @@ private static string GetDefaultMetricsEndpoint()
 
         public static Task<int> Main(string[] args)
         {
-            // FUTURE: This log message should be removed when dotnet-monitor is no longer an experimental tool
-            Console.WriteLine("WARNING: dotnet-monitor is experimental and is not intended for production environments yet.");
+            // Write directly to console so that message is always written regardless of logging level.
+            Console.WriteLine(ExperimentMessage);
```

Comment:
Do we want both? Direct to console and through ILogger?

Reply:
It is also written through ILogger in Startup since I think that is the first place that I can get an ILogger without rebuilding the host to just get the ILogger instance.

Reply:
Yeah, I saw that. My question was why have both?

Reply:
Sorry, read it as "why not have both"? I did it this way so that the warning could not be disabled.

Alternatively, I could put the warning in a separate logging category and then manually add that category through code so that it cannot be disabled. Then we'll just remove the logging call and category when we intend to remove the warning.

Reply:
I now have the message logged through ILogger and it cannot be disabled (its category is always enabled).